packages/cra-external-template/template/src/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { render } from 'react-dom'
+import { Config } from './types/global'
+import App from './core/app'
+import { getMarketplaceGlobalsByKey } from '@reapit/elements'
+import config from './config.json'
```

Comment:
cannot import config directly like this, because it'll cause problem if config.json load after render app